src/main/java/chess/view/validator/PlayValidator.java

```diff
@@ -21,12 +22,13 @@ public void validate(InputRequest inputRequest) {
         next.validate(inputRequest);
     }
 
-    private void validateInput(InputRequest inputRequest) {
-        if (inputRequest.getValue().get(0).equals(END_INPUT)) {
+    private void validateInput(final InputRequest inputRequest) {
+        String input = inputRequest.getValue().get(0);
+        if (input.equals(END_INPUT) || input.equals(STATUS_INPUT)) {
             this.next = new SuccessValidator();
             return;
         }
-        if (inputRequest.getValue().get(0).equals(MOVE_INPUT)) {
+        if (input.equals(MOVE_INPUT)) {
```

Comment:
```suggestion
        if (MOVE_INPUT.equals(input)) {
```
방식으로 `null이아닌것.equals(null이될수있는것)`으로 비교를 해주어야 NPE가 발생할 가능성이 줄어듭니다! :)